Makefile

```diff
@@ -989,7 +989,9 @@ CPP_SOURCES_CCL := $(subst $(PKG_ROOT),$(CPP_PROTO_CCL_ROOT),$(CPP_PROTOS_CCL:%.
 
 UI_PROTOS := $(UI_JS) $(UI_TS)
 
-$(GO_PROTOS_TARGET): $(PROTOC) $(GO_PROTOS) $(GOGOPROTO_PROTO) $(BOOTSTRAP_TARGET) | bin/protoc-gen-gogoroach
+$(GOGOPROTO_PROTO): $(GO_PROTOS)
```

Comment:
Based on your error message, the problem is that $(GOGOPROTO_PROTO) needs to express a dependency on $(SUBMODULES_TARGET). If the fix here worked, I suspect it's only because it tickled Make into evaluating targets in an order that avoided the race.

Reply:
Makes sense. I didn't clean the dir and start from scratch each time, it's possible something had already evaluated in a previous run.